doc/sources/conf.py

```diff
@@ -37,13 +37,13 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Intel(R) Extension for Scikit-learn*'
-copyright = '2022, Intel'
+copyright = '2023, Intel'
 author = 'Intel'
 
 # The short X.Y version
-version = '2023'
+version = '2023.1.1'
```

Comment:
i think intention here was to have more generic major.minor versioning instead of full release version

Reply:
We will have doc for each version (X.Y) or each release (X.Y.Z)?

Reply:
We might have or have not specific doc version for patch(.Z) releases. 

But i think initial intent here was to have a way for talking about particular version and release where this make sense in documentation

Reply:
Now 'version' is shown in left panel of the doc: 
![image](https://github.com/intel/scikit-learn-intelex/assets/88080182/809c8d96-33e3-457b-b79e-7a6bfa91e5aa)

'release' is shown on version selection page:
![image](https://github.com/intel/scikit-learn-intelex/assets/88080182/7aa983c7-02f1-4062-9406-7861ab36f069)

what's your suggestion?

Reply:
use X.Y as version format in https://github.com/intel/scikit-learn-intelex/pull/1323/commits/b66d55aded070c3285a1d12c33bd0e0266dd19db